src/PyCall.jl

```diff
@@ -285,7 +298,9 @@ end
 pyimport(name::AbstractString) =
     PyObject(@pycheckn ccall((@pysym :PyImport_ImportModule), PyPtr,
                              (Cstring,), name))
-
+"""
```

Comment:
I would attach the docstring to `pyimport(name::AbstractString)`, since that is the most common method to call; I may remove the `Symbol` version at some point, so perhaps it is better not to document that.